### PR TITLE
impr(DAPIClient): added DNS lookup for evonet

### PR DESCRIPTION
**Pull request by Alex-Werner**
### Issue being fixed or implemented  

This PR allows to add DNSLookup for Evonet by using the domain name we have set up ('seed.evonet.networks.dash.org')

### What was done  

- deleted Transporter.js
- added utils for DNS lookup (promisification). 
- Transporter.isReady is called when available on Account.isReady (affects time to load)

### How Has This Been Tested?

- added DNS tests.

### Notes  

We should not call that at each time, and should cache it. It will be a next step on this point.
Also, on browser, we do not have dns.lookup support, so we will rely on a hardcoded seed list. 

### Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation

### CODE REVIEW

**Comment by thephez** [comment]
Cool! Looks like the travis failures may not be related to this PR :thinking:

**Comment by Alex-Werner** [comment]
@thephez It is actually, it either requires transporter.isReady() or to pass noLookup:true to transporter. 
Had to update the tests, which is done

File: src/utils/dns.js [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+  { service: '35.166.50.163:3000' },
+  { service: '34.212.69.250:3000' },
+  { service: '54.212.114.204:3000' },
+  { service: '35.166.226.20:3000' },
+  { service: '54.71.60.103:3000' },
+  { service: '34.221.111.92:3000' },
+  { service: '35.166.207.191:3000' },
+  { service: '52.24.203.28:3000' },
+];
+const palinkaSeeds = [
```
Comment (shumkov) [inline src/utils/dns.js:55]: Palinka is a private devnet, we should’t publish it.

File: src/utils/dns.js [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+  '34.214.221.50',
+  '54.213.18.11',
+  '34.211.149.102',
+  '52.38.244.67',
+].map((ip) => ({ service: `${ip}:3000` }));
+
+const lookupHostname = async function lookupHostname(hostname) {
```
Comment (shumkov) [inline src/utils/dns.js:62]: I believe we don’t need to lookup DNS manually. It’s not how it suppose to work. The idea was that you just pass this host to DAPI Client as a seed and that’s it.

**Comment by Alex-Werner** [comment]
Not needed. Done by DAPIClient already.

**Pull request closed without merging**